src/pages/saksbehandling/sakintro/Sakintro.tsx

```diff
@@ -44,9 +44,14 @@ const Sakintro = (props: { sak: Sak }) => {
                                         <div>
                                             <p>Søknads-id: {s.id}</p>
                                             <p>Innsendt: {formatDateTime(s.opprettet, intl)}</p>
-                                            {behandlinger.length === 0 && <p>Status: OPPRETTET </p>}
+                                            {behandlinger.length === 0 && !s.søknadTrukket && <p>Status: OPPRETTET</p>}
+                                            {s.søknadTrukket && (
```

Comment:
på linje 47 checkar du om det är `!s.søknadTrukket` och på linje 48 så har du `s.søknadTrukket && ...`. Det kommer ju aldrig ske för både `!s.søknadTrukket && s.søknadTrukket` kan aldrig vara sant.

Reply:
Litt usikker på hva du mener. 
s.søknadTrukket (renamet til lukket etter etter litt jobb i backend) er en Nullable<Lukket>. Hvis den er null, er søknad aktiv, og vi viser status OPPRETTET (fra det jeg fant ut, er dette bare en frontend ting, og ikke en faktisk status fra backend). Hvis den er lukket, viser vi ikke status.

Hvis den ikke er null, Viser vi at denne søknaden er avsluttet. De sjekker for 2 forskjellige ting,.